src/macaron/config/defaults.ini

```diff
@@ -345,8 +347,13 @@ builder =
 # Build args not defined since npm build is just a plumbing command https://docs.npmjs.com/cli/v6/commands/npm-build
 # and SLSA v1.0 removes the scripted build requirement https://slsa.dev/spec/v1.0/requirements
 build_arg =
+    install
+build_run_arg =
+    build
```

Comment:
I think this comment is now not relevant with the new changes to the NPM build tool:
- `install` is treated as `build_arg`
- new addition of `build_run_arg` and `deploy_run_arg`.

Reply:
See commit f0d89a961fe4380ceeb89526ff1e7cbacd680294